editoast/src/views/paced_train.rs

```diff
@@ -446,6 +463,32 @@ mod tests {
         assert_eq!(response.len(), 1);
     }
 
+    #[rstest]
+    async fn update_paced_train() {
+        let app = TestAppBuilder::default_app();
+        let pool = app.db_pool();
+
+        let timetable = create_timetable(&mut pool.get_ok()).await;
+        let paced_train = create_simple_paced_train(&mut pool.get_ok(), timetable.id).await;
+
+        let mut paced_train_base = simple_paced_train_base();
+        paced_train_base.paced.duration = Duration::minutes(90).try_into().unwrap();
+        paced_train_base.paced.step = Duration::minutes(15).try_into().unwrap();
+
+        let request = app
+            .put(format!("/paced_train/{}", paced_train.id).as_str())
+            .json(&json!(&paced_train_base));
+
+        let response: PacedTrainResult =
+            app.fetch(request).assert_status(StatusCode::OK).json_into();
+
+        assert_eq!(
+            response.paced_train.paced.duration,
+            paced_train_base.paced.duration
+        );
+        assert_eq!(response.paced_train.paced.step, paced_train_base.paced.step);
```

Comment:
A stronger check would retrieve the updated PacedTrain from the DB and check that it equals to `paced_train_base`. With such tests it's easy to break the endpoint while still making the test pass.